docs/installation/mac.md

```diff
@@ -51,48 +61,57 @@ To set up your environment:
 
 Getting and building from source:
 
-1. Use `git clone` to to check out a copy of HELICS.
+1. Use `git clone` to check out a copy of HELICS.
 2. Create a build folder. Run cmake and give it the path that HELICS
    was checked out into.
 
-```bash
-git clone https://github.com/GMLC-TDC/HELICS
-cd HELICS
-mkdir build
-cd build
-```
+    ```bash
+    git clone https://github.com/GMLC-TDC/HELICS
+    cd HELICS
+    mkdir build
+    cd build
+    ```
 
 ## Compile and Install
 
-There are a number of different options and approaches at this point depending on your needs, in particular with respect to programming language support:
+There are a number of different options and approaches at this point depending on your needs, in particular with respect to programming language support.
 
-### Basic Install (without language bindings)
+<div class="admonition note">
 
-1. Run `make`.
+Note: For any of these options, if you want to install in a custom location, you can add the following CMake argument: `-DCMAKE_INSTALL_PREFIX=/path/to/install/folder/`
```

Comment:
Add this line:

"There are also a number of other options, and you can check them out by running `ccmake .` in the `build` folder."